common/js/mastra/agent.ts

```diff
@@ -0,0 +1,42 @@
+import { Agent, Memory } from '@mastra/core';
+import { openai } from '@ai-sdk/openai';
+import { z } from 'zod';
+
+const simpleWorkflow = {
+  start: () => ({
+    llm: {
+      messages: [
+        { role: 'system', content: 'You are a helpful assistant.' },
+        { role: 'user', content: 'Hello, who are you?' }
+      ],
+      model: openai('gpt-4o-mini'),
+    },
+    then: (response) => {
+      console.log('Agent response:', response.content);
+      return { result: response.content };
+    }
+  })
+};
+
+const memory = new Memory();
+
+const agent = new Agent({
+  name: '{{ .AgentName }}',
+  instructions: 'You are a helpful assistant that provides concise and accurate information.',
+  model: openai('gpt-4o-mini'),
+  memory,
+  workflow: { simpleWorkflow }
+});
+
+async function runAgent() {
```

Comment:
to be an agentuity agent, you must export a function that implements AgentHandler.

See https://github.com/agentuity/sdk-js/blob/main/src/types.ts.

The agent execution for Mastra should be inside this AgentHandler function.

The most simple Agent example is: https://github.com/agentuity/templates/blob/main/common/js/agent.ts

For logging, make sure to use ctx.logger instead of console.log when possible.